### PR TITLE
feat: add yq executable to both builders

### PR DESCRIPTION
**Pull request by Unix4ever**
Add to both for consistency.

### CODE REVIEW

**Review by rsmitty — APPROVED** [review]

File: Dockerfile.actions-runner-ubuntu-22.04 [hunk 1]
```diff
@@ -1,6 +1,7 @@
 FROM summerwind/actions-runner-dind:ubuntu-22.04
 # renovate: datasource=github-releases depName=google/go-containerregistry
 ARG CRANE_VERSION=v0.16.1
+ARG YQ_VERSION=v4.40.5
```
Comment (frezbo) [inline Dockerfile.actions-runner-ubuntu-22.04:4]: also add the renovate comment

Comment (frezbo) [reply Dockerfile.actions-runner-ubuntu-22.04:4]: the actions runner has some extra repos configured, so sometimes just adding `yq` to list of packages would work

Comment (Unix4ever) [reply Dockerfile.actions-runner-ubuntu-22.04:4]: > the actions runner has some extra repos configured, so sometimes just adding yq to list of packages would work

tried that, doesn't seem to work

Comment (frezbo) [reply Dockerfile.actions-runner-ubuntu-22.04:5]: ahh, then no issues :+1:

**Pull request merged**